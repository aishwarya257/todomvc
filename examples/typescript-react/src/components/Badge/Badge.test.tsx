import React from 'react';
import {render, screen} from '@testing-library/react';
import Badge from './Badge';

describe('test Badge component', () => {
    it('render Badge component', () => {
        const text = 'I am a sample test';
        render(<Badge title={text}>{text}</Badge>);
        expect(screen.queryByText(text)).toBeInTheDocument();
    });
    it('create snapshot', () => {
        const anotherText = 'I am a text';
        const {container} = render(<Badge title={anotherText}>{anotherText}</Badge>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
