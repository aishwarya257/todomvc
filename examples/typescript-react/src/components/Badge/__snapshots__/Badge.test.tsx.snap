// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Badge component create snapshot 1`] = `
<span
  class="badge"
  title="I am a text"
>
  I am a text
</span>
`;
